core/code/map_data_render.js

```diff
@@ -106,7 +106,7 @@ window.Render.prototype.processDeletedGameEntityGuids = function(deleted) {
 
 }
 
-window.Render.prototype.processGameEntities = function(entities) {
+window.Render.prototype.processGameEntities = function(entities, details) {
```

Comment:
`processGameEntities` could be call only when we get entites (`getEntities` or `mapInject` hook) and remove the `details` parameters

Reply:
```suggestion
window.Render.prototype.processGameEntities = function(entities) {
```